packages/vitest-pool-workers/package.json

```diff
@@ -67,19 +67,19 @@
 		"@cloudflare/workers-types": "^4.20240909.0",
 		"@types/node": "20.8.3",
 		"@types/semver": "^7.5.1",
-		"@vitest/runner": "1.3.x - 1.5.x",
-		"@vitest/snapshot": "1.3.x - 1.5.x",
+		"@vitest/runner": "catalog:default",
+		"@vitest/snapshot": "catalog:default",
 		"capnp-ts": "^0.7.0",
 		"capnpc-ts": "^0.7.0",
 		"ts-dedent": "^2.2.0",
 		"typescript": "^5.5.2",
 		"undici": "^5.28.4",
-		"vitest": "1.5.0"
+		"vitest": "catalog:default"
 	},
 	"peerDependencies": {
-		"@vitest/runner": "1.3.x - 1.5.x",
-		"@vitest/snapshot": "1.3.x - 1.5.x",
-		"vitest": "1.3.x - 1.5.x"
+		"@vitest/runner": "2.0.x",
+		"@vitest/snapshot": "2.0.x",
+		"vitest": "2.0.x"
```

Comment:
These don't use the catalog because I'm hopeful we can make it a range in future